runtimes/pythonrt/pythonrt.go

```diff
@@ -468,6 +468,10 @@ func (py *pySvc) initialCall(ctx context.Context, funcName string, args []sdktyp
 				cb.successChannel <- val
 			}
 		case v := <-py.channels.done:
+			pCtx, cancel := context.WithTimeout(ctx, 3*time.Second)
```

Comment:
maybe we should check if the process is still up instead of counting on time ?
3 seconds would probably work most of the times, but we saw many problems when system is under load
so I guess it might not really work sometimes

Reply:
I thought of something like:
```go
	ctx, cancel := context.WithCancel(ctx)
	defer cancel()
	go func() {
		var req pb.HealthRequest
		for range time.Tick(100 * time.Millisecond) {
			if _, err := py.runner.Health(ctx, &req); err != nil {
				cancel()
			}
		}
	}()
```

What do you think?

Reply:
Leaving at 1 sec wait. We'll see...